m/configDataPreprocessingSeaexplorer.m

```diff
@@ -0,0 +1,129 @@
+function preprocessing_options = configDataPreprocessingSeaexplorer()
+%CONFIGDATAPREPROCESSINGSEAEXPLORER  Configure Seaexplorer glider data preprocessing.
+%
+%  Syntax:
+%    PREPROCESSING_OPTIONS = CONFIGDATAPREPROCESSINGSEAEXPLORER()
+%
+%  Description:
+%    PREPROCESSING_OPTIONS = CONFIGDATAPREPROCESSINGSEAEXPLORER() should return 
+%    a struct setting the options for Seaexplorer glider data preprocessing 
+%    as needed by the function PREPROCESSGLIDERDATA.
+%
+%  Examples:
+%    preprocessing_options = configDataPreprocessingSeaexplorer()
+%
+%  See also:
+%    PREPROCESSGLIDERDATA
+%
+%  Authors:
+%    Frederic Cyr  <Frederic.Cyr@mio.osupytheas.fr>
+%    Joan Pau Beltran  <joanpau.beltran@socib.cat>
+
+%  Copyright (C) 2013-2015
+%  ICTS SOCIB - Servei d'observacio i prediccio costaner de les Illes Balears
+%  <http://www.socib.es>
+%
+%  This program is free software: you can redistribute it and/or modify
+%  it under the terms of the GNU General Public License as published by
+%  the Free Software Foundation, either version 3 of the License, or
+%  (at your option) any later version.
+%
+%  This program is distributed in the hope that it will be useful,
+%  but WITHOUT ANY WARRANTY; without even the implied warranty of
+%  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%  GNU General Public License for more details.
+%
+%  You should have received a copy of the GNU General Public License
+%  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+  error(nargchk(0, 0, nargin, 'struct'));
+  
+  preprocessing_options = struct();
+
+  %% Navigation
+  preprocessing_options.time_list(1).time = 'Posixtime_sci';
+  preprocessing_options.position_list(1).longitude = 'NAV_LONGITUDE';
+  preprocessing_options.position_list(1).latitude  = 'NAV_LATITUDE';
+  preprocessing_options.position_list(1).conversion = @nmea2deg;
+  
+  %preprocessing_options.depth_list(1).depth = 'NAV_DEPTH';
+  preprocessing_options.depth_list(1).depth = 'GPCTD_PRESSURE';
+  preprocessing_options.depth_list(2).depth = 'SBD_PRESSURE';
+  %preprocessing_options.depth_list(3).depth = 'NAV_DEPTH';
+
```

Comment:
Indeed, I was a bit messed with the `depth` and `depth_ctd` variables. 

In SeaExplorer, only ctd_pressure is used. So `Depth` and `NAV_DEPTH` are the same except one is sub-sampled and found in navigation files (`Depth` ) while the other is found in the higher sampling rate science files